__tests__/app.test.js

```diff
@@ -224,4 +224,85 @@ describe("express app", () => {
       });
     });
   });
+  describe('GET /api/articles', () => {
+    it('status 200: with array of articles', async () => {
+      const {body: {articles}} = await request(app).get("/api/articles")
+      expect(articles).toEqual(expect.arrayContaining([
```

Comment:
Off the top of my head, I think arrayContaining would possibly still pass if there were elements included inside of it that were not the objects you expected to find. Also it will only check a single object if the first object it finds matches what we are expecting